scripts/verify.sh

```diff
@@ -163,7 +163,7 @@ source $SCRIPTDIR/common.bash
 
 header
 log separator
-verify_min_k8s_version "1" "18"
+verify_min_k8s_version "1" "20"
```

Comment:
Please revert this change as currently there are drivers that refer to this version in config.yaml